gr-azure-software-radio/README.md

```diff
@@ -55,27 +55,31 @@ Run the QA tests with any of the following methods:
  - Or from the python directory:
     ```
     python -m pytest qa_*
-    ```    
+    ```
 
 ### Running the Integration Tests
-To run the integration tests for the blob blocks, you must first create a storage account on Azure 
-and obtain the connection string. Next, export this connection string into an environment variable 
-named "AZURE_STORAGE_CONNECTION_STRING". 
+To run the integration tests for the blob blocks, you must first create a storage account on Azure
```

Comment:
we need to update the Table of Contents "running the tests" to link to the unit test and integration test sections

Reply:
This was a great catch btw